calibre.json

```
@@ -0,0 +1,64 @@
+{
+  "Calibre": {
+    "description": "Calibre is an open source e-Book management program.<p>Calibre allows for the management of e-Books and syncing across devices using the content server.</p><p>For more information visit <a href='https://calibre-ebook.com/' target='_blank' rel='noopener noreferrer'>Calibre's Official Site</a> and <a href='https://hub.docker.com/r/linuxserver/calibre' target='_blank' rel='noopener noreferrer'>the Docker container page.</a></p>",
+    "more_info": "",
+    "version": "0.0.1",
+    "website": "https://calibre-ebook.com/",
+    "ui": {
+      "slug": ""
+    },
+    "volume_add_support": true,
+    "containers": {
+      "calibre": {
+        "image": "linuxserver/calibre",
+        "launch_order": 1,
+        "opts": [
+          [
+            "-v",
+            "/config"
+          ]
+        ],
+        "ports": {
+          "8080": {
+            "description": "Calibre WebUI port.",
+            "host_default": 8080,
+            "label": "WebUI port",
+            "protocol": "tcp",
+            "ui": true
+          },
+          "8081": {
+            "description": "Calibre Server port.",
+            "host_default": 8081,
+            "label": "Server Port",
+            "protocol": "tcp",
+            "ui": true
```

It seems only the port `8080` is used for the webUI, so change this to `false`:
```suggestion
            "ui": false
```